README.md

```diff
@@ -13,11 +15,8 @@ This repository contains template projects for Vehicle Damage Modeling project.
 ### Structure
 There are two projects:
 
- - _VehicleDamageModeling_ - template for developing framework with UI component for choosing damaged parts of vehicle
- - _SampleApp_ - sample application which demonstrates how to import and use _VehicleDamageModeling_ framework
-
-### Notice
-iOS project uses [Metal Framework](https://developer.apple.com/metal/) for rendering. As some of required classes (like `CAMetalLayer`) is not available on iOS Simulator - please use iOS Device as compilation target.
+ - _FVM_ - developing framework with UI component for choosing damaged parts of vehicle
```

Comment:
developing framework with UI component for choosing -> framework containing UI component that allows user to choose